README.md

```diff
@@ -1066,23 +1057,23 @@ For example:
 * `delete({'a' => 1,'b' => 2,'c' => 3},['b','c'])` returns {'a'=> 1}.
 * `delete(['ab', 'b'], 'b')` returns ['ab'].
 
-Note that from puppet 4.0.0 the minus (`-`) operator deletes values from arrays and keys from a hash:
+Since Puppet 4.0.0, the minus (`-`) operator deletes values from arrays and deletes keys from a hash:
 
     ['a', 'b', 'c', 'b'] - 'b'
     # would return ['a', 'c']
 
     {'a'=>1,'b'=>2,'c'=>3} - ['b','c'])
     # would return {'a' => '1'}
 
-A global delete from a string can be performed with the built-in
+Perform a global delete from a string with the built-in
 [`regsubst`](https://puppet.com/docs/puppet/latest/function.html#regsubst) function.
 
     'abracadabra'.regsubst(/bra/, '', 'G')
     # would return 'acada'
 
 In general, the built-in
 [`filter`](https://puppet.com/docs/puppet/latest/function.html#filter) function
-can filter out entries from arrays and hashes based on keys and/or values.
+can filter out entries from arrays and hashes based on keys and values.
```

Comment:
Not quite correct, it is keys (only), values (only) or keys and values. If you do not like "and/or", suggest saying "based on a combination of keys and values".